app.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require './lib/chitter.rb'
+require './lib/identity.rb'
+class ChitterBox < Sinatra::Base
+  get '/' do
+    erb :sign_in
+  end
+
+  post '/' do
+    redirect '/'
+  end
+
+  get '/posts' do
+    @username = $identity.name
```

Comment:
watch out for global variables, even if it's a temporary measure maybe better to store them in sessions instead